packages/destination-actions/src/destinations/actions-facebook-conversions/constants.ts

```diff
@@ -0,0 +1,181 @@
+export const CURRENCY_ISO_CODES = [
```

Comment:
If you use `Set` instead of `Array` you will get a minor perf improvement when calling `.has` vs `.includes` (assuming integration is handling large throughput of events this will be worth it -- https://egghead.io/lessons/javascript-use-es6-sets-to-improve-javascript-performance )


```suggestion
export const CURRENCY_ISO_CODES = new Set([
```

Reply:
Good call, we can also use the set for the GA4 constant.ts as well

Reply:
Actually, GA4 and FB CAPI are going to both use this same set. Would it make sense to pull the `constants.ts` out into /lib so they can both import from the same file. And later on if other destinations need to check currency it can be reused

Reply:
I would hesitate to introduce a cross-destination dependency. IMO we should keep destinations as isolated as possible.